mars/core/graph/tests/test_graph.py

```diff
@@ -105,3 +108,75 @@ def test_to_dot():
 
     dot = to_str(graph.to_dot(trunc_key=5))
     assert all(to_str(n.key)[:5] in dot for n in graph) is True
+
+
+def test_tileable_graph_logic_key():
+    # Tensor
+    t1 = mt.random.randint(10, size=(10, 8), chunk_size=4)
+    t2 = mt.random.randint(10, size=(10, 8), chunk_size=5)
+    graph1 = (t1 + t2).build_graph(tile=False)
+    graph2 = (t1 + t2).build_graph(tile=False)
```

Comment:
Is the logic key remains the same even the tileable instances are different?
```python
graph1 = (t1 + t2).build_graph(tile=False)
tt1 = mt.random.randint(10, size=(10, 8), chunk_size=4)
tt2 = mt.random.randint(10, size=(10, 8), chunk_size=5)
graph2 = (tt1 + tt2).build_graph(tile=False)
assert graph1.logic_key == graph2.logic_key
```

Reply:
Yes. Your writing is better, I'll optimize this.

Reply:
Good to know this.